### PR TITLE
docs: templates update v2

### PR DESCRIPTION
**Pull request by zmzlois**
# What's changed in this PR

- Remove *Creating workspace from templates* in templates guide
- Add templates index explaination and folder structure
- Add custom script to update templates

### CODE REVIEW

File: src/content/docs/configuration/templates.mdx [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+<hr/>
+
+## Default templates index
+
+The default template index is a JSON file that contains an array of 13 objects. Each object represents a template. The template is located in [this repository](https://github.com/daytonaio/samples-index/blob/main/index.json).
+
+{/* <!-- TODO: try template created in one repo and see if the problem with setting remote origin URL persists.  --> */}
+
+
+Below is an indexed description of the default templates, their content and repository URLs.
```
Comment (ivan-burazin) [inline src/content/docs/configuration/templates.mdx:28]: @zmzlois please remove the list of current templates. non important

File: src/content/docs/configuration/templates.mdx [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+</TabItem>
+</Tabs>
+
+:::note
+After the deployment, you might need to do a hard refresh on your browser (using <Keyboard shortcuts={['Ctrl', 'Shift', 'R']}/>) to see the changes.
+:::
+
+
+
+### Deploy with Terraform
```
Comment (ivan-burazin) [inline src/content/docs/configuration/templates.mdx:454]: @zmzlois remove everything connected to Terraform

**Review by ivan-burazin — CHANGES_REQUESTED** [review]
Comments what needs to be removed

**Comment by zmzlois** [comment]
removed, @nkkko

**Review by ivan-burazin — APPROVED** [review]

**Pull request merged**